.eslintrc

```diff
@@ -0,0 +1,52 @@
+{
+  "extends": [
+    "lisk-base",
+    "airbnb-base"
```

Comment:
This is not necessary as `lisk-base` already extends `airbnb-base`.
https://github.com/LiskHQ/eslint-config-lisk-base/blob/master/index.js